csrc/preprocess/transform/default_format_bundle.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) OpenMMLab. All rights reserved.
+
+#include "default_format_bundle.h"
+
+#include <cassert>
+
+#include "archive/json_archive.h"
+#include "core/tensor.h"
+
+namespace mmdeploy {
+
+DefaultFormatBundleImpl::DefaultFormatBundleImpl(const Value& args) : TransformImpl(args) {
+  if (args.contains("img_to_float") && args["img_to_float"].is_boolean()) {
+    arg_.img_to_float = args["img_to_float"].get<bool>();
+  }
+  if (args.contains("pad_val") && args["pad_val"].is_object()) {
+    for (auto& [key, _] : arg_.pad_val)
+      if (args["pad_val"].contains(key) && args["pad_val"][key].is_number_float()) {
+        arg_.pad_val[key] = args["pad_val"][key].get<float>();
+      }
+  }
+}
+
+Result<Value> DefaultFormatBundleImpl::Process(const Value& input) {
+  MMDEPLOY_DEBUG("DefaultFormatBundle input: {}", to_json(input).dump(2));
+  Value output = input;
+  if (input.contains("img")) {
+    Tensor tensor = input["img"].get<Tensor>();
+    OUTCOME_TRY(output["img"], ToFloat32(tensor, arg_.img_to_float));
+  }
+
+  Tensor tensor = output["img"].get<Tensor>();
+  // set default meta keys
```

Comment:
I don't think the default key should be set here. 

If `Resize` goes after `DefaultFormatBundle` in the preprocessing pipeline, the `scale_factor` generated by `DefaultFormatBundle` will shadow the if-else branch `arg_.img_scale`  of  `Resize`.